helm/values.yaml

```diff
@@ -211,6 +211,8 @@ configMap:
               - processedSpans
               - queueSize
               - ^otlp.*
+              - ^mirror_stats_.*
+              - traceableagent_heartbeat_bucket
```

Comment:
Should we exclude this one as well?
`otelcol_processor_batch_batch_send_size_bucket `

Are we using this for any active monitoring/alerting?

cc: @tim-mwangi

Reply:
Yes. We can include `otelcol_processor_batch_batch_send_size_bucket` for now since we do not have any active alerts on it.

Reply:
done.